### ISSUE TITLE
bitcoin transaction

### ISSUE THREAD

**Issue by mix0nSPb** [DarrenWestwood/greed#44]
Hello! Today I decided to test your wonderful script for finalizing the bot for paying for bitcoin, I really liked it and I put it on my server, according to the instructions, everything works fine! only one problem arose for me, when I replenished my account in the bot, the transaction went through, the bot received a notification of successful payment, but the balance itself was not replenished, as it was, and remained 0 euros ... can you tell me what could be the problem?

**Comment by mix0nSPb** [comment]
Yes, I understand you very well that the funds should be credited after 2 confirmations! but here is the problem:
I installed the bot according to the instructions, it started working fine, I decided to try to top up the balance for 5 euros, made the first transaction from one account, the second transaction also for 5 euros, then I logged in from the second account and made the third transaction for 5 euros. [screenshot](https://imgur.com/a/opTaXb6) 2 transactions on the 1st account are marked in yellow, the last transaction on the next day on the 2nd account is marked in red.
all transactions have been confirmed by the system for a long time, since all funds came to my wallet in Blockchain.
But still, when I log into both accounts, they have a balance of 0 euros... [Screenshot](https://imgur.com/a/2nEj16T)
And in both accounts, I received only one notification like this: [screenshot] (https://imgur.com/a/T54EdiP) the second notification, like yours, did not come to me in any account!

**Comment by DarrenWestwood** [comment]
Since the 2nd notification is missing from the chat logs, it appears there may have been an issue with the confirmed callback requests reaching your bot. 
Please check your [Blockonomics logs](https://www.blockonomics.co/merchants#/logs) to verify the _Confirmed_ callbacks reached your bot correctly. The response code in the logs should be 200 to indicate the callback request was received by your bots callback URL.

**Comment by omarnazih** [comment]
Did you solve the issue yet as i'm facing it too!

**Comment by DarrenWestwood** [comment]
Thanks @omarnazih for letting me know this is functioning correctly with the latest version.
